packages/dialog/src/vaadin-lit-dialog.js

```diff
@@ -0,0 +1,169 @@
+/**
+ * @license
+ * Copyright (c) 2017 - 2023 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import './vaadin-lit-dialog-overlay.js';
+import { css, html, LitElement } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
+import { OverlayClassMixin } from '@vaadin/component-base/src/overlay-class-mixin.js';
+import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
+import { ThemePropertyMixin } from '@vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
+import { DialogBaseMixin } from './vaadin-dialog-base-mixin.js';
+import { DialogDraggableMixin } from './vaadin-dialog-draggable-mixin.js';
+import { DialogResizableMixin } from './vaadin-dialog-resizable-mixin.js';
+
+export { DialogOverlay } from './vaadin-lit-dialog-overlay.js';
+
+/**
+ * LitElement based version of `<vaadin-dialog>` web component.
+ *
+ * ## Disclaimer
+ *
+ * This component is an experiment not intended for publishing to npm.
+ * There is no ETA regarding specific Vaadin version where it'll land.
+ * Feel free to try this code in your apps as per Apache 2.0 license.
+ *
+ * @extends HTMLElement
+ * @mixes ElementMixin
+ * @mixes DialogBaseMixin
+ * @mixes DialogDraggableMixin
+ * @mixes DialogResizableMixin
+ * @mixes OverlayClassMixin
+ * @mixes ThemePropertyMixin
+ */
+class Dialog extends DialogDraggableMixin(
+  DialogResizableMixin(DialogBaseMixin(OverlayClassMixin(ThemePropertyMixin(ElementMixin(PolylitMixin(LitElement)))))),
+) {
+  static get is() {
+    return 'vaadin-dialog';
+  }
+
+  static get styles() {
+    return css`
+      :host {
+        display: none !important;
+      }
+    `;
+  }
+
+  static get properties() {
+    return {
+      /**
+       * Set the `aria-label` attribute for assistive technologies like
+       * screen readers. An empty string value for this property (the
+       * default) means that the `aria-label` attribute is not present.
+       */
+      ariaLabel: {
```

Comment:
These are duplicated because we have no component-specific mixin for `vaadin-dialog`, only a base mixin that is also used by other dialog components where we don't need these properties?

Also asking because I want to know if there is a need to declare properties differently for Polymer and Lit, or if `PolyLitMixin` can handle there differences.

Reply:
I was thinking to add `DialogRendererMixin` where these should be declared. Let me do that.

>  there is a need to declare properties differently for Polymer and Lit

`PolylitMixin` covers differences between Lit and Polymer by supporting `value`, `readOnly`, `notify`, `observer` and `computed` in the properties block.

Reply:
Created https://github.com/vaadin/web-components/pull/6188 to reduce duplication of renderer properties.

Reply:
Looks good, thanks.